.rubocop.yml

```diff
@@ -101,34 +101,40 @@ RSpec:
       - expect_no_offenses
       - expect_offense
 
+RSpec/DescribeClass:
+  Exclude:
+    - spec/project/**/*.rb
+
 RSpec/ExampleLength:
   CountAsOne:
     - heredoc
   Max: 11
 
-RSpec/DescribeClass:
-  Exclude:
-    - spec/project/**/*.rb
+RSpec/ExpectInLet:
+  Enabled: true
```

Comment:
I’m not sure where this entry comes from, ExpectInLet is already enabled by default.yml 😅 

I’ll open a separate pull request with the other changes to .rubocop.yml.

Reply:
Done: #1958